distribution/tools/entitlement-trampoline/README.md

```diff
@@ -0,0 +1,12 @@
+### Entitlement Trampoline
+
+This is the code called directly from instrumented methods.
+It's a minimal code stub that is loaded into the boot classloader by the entitlement agent
+so that it is callable from the class library methods instrumented by the agent.
+Its job is to forward the entitlement checks to the actual runtime library,
+which is loaded normally.
+
+This module is responsible for defining which class library methods are to be instrumented.
```

Comment:
It does not look like the right place to me, but we can see. For now, a simple hardcoded map in the agent with one entry would be more than enough.